lib/jsdom/browser/htmltodom.js

```diff
@@ -237,7 +267,26 @@ function setChild(core, parent, node) {
     return null;
   }
 
-  newNode[locationInfo] = node.__location;
+  if (node.__location) {
+    newNode[locationInfo] = {
```

Comment:
Not too sure about this. This is mainly needed for backwards compat because start/end got renamed to (start|end)Offset. We could just alias them, or major ourselves...